content/lessons/02_ownership/index.md

```diff
@@ -344,19 +368,20 @@ fn main() {
   count_animals(1, &s);
   count_animals(2, &s);
   count_animals(3, &s);
+  count_animals(4, "goat"); // In this version of the code it doesn't compile.
 }
 ```
 
 ### Further reading
 
-- [Char documentation](https://doc.rust-lang.org/std/primitive.char.html)
+- [The Book, chapter 4](https://doc.rust-lang.org/stable/book/ch04-00-understanding-ownership.html) - obligatory reading
 
-- [Working with strings in Rust](https://fasterthanli.me/articles/working-with-strings-in-rust)
+- [Rust by Example, chapter Scoping rules](https://doc.rust-lang.org/stable/rust-by-example/index.html) - obligatory reading
```

Comment:
♻️ Instead of annotating each link with `obligatory/additional reading`, let's have two subsections.